### DISCUSSION TITLE
Feature request: update vscode version specified under engines

### DISCUSSION THREAD

**Discussion by aosen-xiong** [renovatebot/renovate#24365] [Suggest an Idea]
### Tell us more.

I noticed `engines: Renovate will update any node, npm and yarn version specified under engines.` in your docs. Can you also support updating vscode version under engines and make it consistent with dependencies? Thanks.
```json
"engines": {
    "vscode": "^1.82.0"
  },
  "devDependencies": {
    "@types/vscode": "^1.82.0",
  },
```

**Comment by rarkins** [comment]
I think it may already do

**Reply by XoL1507** [reply]
sorry

**Reply by aosen-xiong** [reply]
I just tried in my own folk. Looks like it does not update the engines and it is odd to only update the package-lock.json file. https://github.com/Ao-senXiong/checker-framework-vscode/pull/3

**Reply by rarkins** [reply]
Unless a v2 comes out, there is no need to update the range you already have declared in package.json

**Reply by aosen-xiong** [reply]
Thanks, sometimes I got this ERROR  @types/vscode ^1.82.0 greater than engines.vscode ^1.81.0. That's why I am think about upgrading them both.

**Reply by rarkins** [reply]
I suggest you pin them both to exact versions of it matters

**Reply by aosen-xiong** [reply]
Thanks!
